src/firefly/js/core/AppDataCntlr.js

```diff
@@ -211,7 +211,7 @@ export function getConnectionCount(channel) {
  * @param wp center WorldPt
  * @param corners array of 4 WorldPts that represent the corners of a image
  */
-export const dispatchActiveTarget= function(wp,corners) {
+export const dispatchActiveTarget= function(wp,corners=undefined) {
```

Comment:
is this for documentation purposes?  it can only be undefined when it's undefined.

Reply:
yes, I stared doing this more because is kind of self documents.